### ISSUE TITLE
numbering in create_test output

### ISSUE THREAD

**Issue by mnlevy1981** [ESMCI/cime#1176]
When I run `create_test`, I tend to leave the output in a terminal window and glance at it periodically to see when it finishes. Current output includes lines like

```
Starting SHAREDLIB_BUILD for test ERI.T62_g16.CECO.yellowstone_intel.pop-ciso with 1 procs
Starting RUN for test ERS.T62_g37.CECO.yellowstone_intel.pop-ciso_transient with 1 proc on interactive node and 30 procs on compute nodes
Finished RUN for test ERS.T62_g37.CECO.yellowstone_intel.pop-ciso_transient in 7.619958 seconds (PEND)
Finished SHAREDLIB_BUILD for test ERI.T62_g16.CECO.yellowstone_intel.pop-ciso in 22.604582 seconds (PASS)
```

But seeing something like

```
Starting SHAREDLIB_BUILD for test ERI.T62_g16.CECO.yellowstone_intel.pop-ciso with 1 procs
Starting RUN for test ERS.T62_g37.CECO.yellowstone_intel.pop-ciso_transient with 1 proc on interactive node and 30 procs on compute nodes
Finished RUN for test ERS.T62_g37.CECO.yellowstone_intel.pop-ciso_transient in 7.619958 seconds [TEST X of Y] (PEND)
Finished SHAREDLIB_BUILD for test ERI.T62_g16.CECO.yellowstone_intel.pop-ciso in 22.604582 seconds (PASS)
```

where `X` is a running count and `Y` is the number of tests being created would be useful in knowing how far along the test suite is.

Relatedly, I noticed the `cs.status` script isn't created until the very end of `create_test` -- is there a reason for this? In previous CESM versions the script was generated immediately and then I didn't need to watch the `create_test` output because I could see how many tests were left to build by parsing the status output. I thought about making a separate issue ticket for this, and am happy to do so if warranted.

**Comment by jedwards4b** [comment]
Tests are run in parallel so seeing test 4 of 8 completed a given phase is probably not a good indication that tests 1-3 have also completed that phase or that tests 5-8 have not.

**Comment by mnlevy1981** [comment]
So each parallel task writes to stdout without communicating with the other tasks? I was just hoping for a running count of the number of tests that have been successfully built and submitted to the queue but I guess I could filter the results of `qstat` or `bjobs` and count by hand.

Any thoughts on generating the `cs.status` script at the beginning of `create_test` instead of the end?

**Comment by jgfouca** [comment]
@mnlevy1981 I can look into improving the output from create_test. It shouldn't be a problem to generate cs.status much earlier.
